### PR TITLE
Yast

### PR DESCRIPTION
**Pull request by jsegitz**

### CODE REVIEW

File: 2018/20xxx/CVE-2018-20106.json [hunk 1]
```diff
@@ -1,18 +1,94 @@
 {
```
Comment (cve-team) [inline 2018/20xxx/CVE-2018-20106.json]: u'vendor_name' is a required property

**Comment by iamleot** [comment]
Hello Johannes, 

Johannes Segitz writes:
> [...]
>   * data for CVE-2018-17955
> [...]

In CVE-2018-17955 - by reading the description - it is not clear which
software is affected.  Can you please add:

 `In yast-multipath before version 4.1.1 static [...]'


Thank you!

**Comment by cve-team** [comment]
Hi,

Please include the affected product name and affected versions in the descriptions for CVE ids CVE-2018-17956 and CVE-2018-20106.

Thank you.

**Comment by cve-team** [comment]
Hi, 

CVE-2018-20106 states in the "version" container that the affected versions of yast2-printer includes version 4.0.2, but the description says that it affects versions before 4.0.2. I wasn't able to resolve this discrepancy with the reference you provided. Can you clarify if version 4.0.2 is affected?

Thank you.

**Comment by cve-team** [comment]
Hi,

For CVE-2018-17955, could you please change the name of the product in the description so that it matches the product name in the "affects" container?

For CVE-2018-17956, I get a "You are not authorized to access bug #1117597." message after successfully login in to a free user account I created. Could you please provide another reference for this CVE id or make this reference publicly available? 

Thank you.

**Comment by jsegitz** [comment]
I made 1117597 public, should be accessible to you now

**Comment by cve-team** [comment]
Thank you. We have populated the entry for CVE-2018-17955 CVE-2018-17956 CVE-2018-20106 in the CVE List. It should be published on https://cve.mitre.org/ in the next hour or two.

CVE Team

**Pull request merged**